tests/www/test_security.py

```diff
@@ -235,8 +244,17 @@ def test_access_control_with_invalid_permission(self):
             'can_varimport',  # a real permission, but not a member of DAG_PERMS
             'can_eat_pudding',  # clearly not a real permission
         ]
+        username = "Mrs. User"
```

Comment:
Do we allow spaces in username ?

Reply:
Not sure. But for the sake of the test it doesn't impact behavior.